test/journeys/lib/test-helpers/basic.js

```diff
@@ -0,0 +1,28 @@
+export const elements = {
```

Comment:
oooh you're right we aren't using it anywhere else. thanks for clarifying, i'll make these changes now!